packages/builder/src/builder.js

```diff
@@ -447,7 +447,7 @@ export default class Builder {
               dst: t.dst || path.basename(t.src || t),
               custom: true
             },
-            t
+            typeof t === 'object' ? t : undefined
```

Comment:
Isn't better-throwing errors instead of silent pass?

Reply:
I think it's a correct behaviour, t can be String here, like 

```js
// nuxt.config.js

build: {
  templates: [
    'foo.js'
  ]
}
```

Reply:
NVM. Didn't noticed it is passed as second arg.